src/sass/_flourish-pullquotes.scss

```diff
@@ -0,0 +1,68 @@
+.flourish-pullquote-wrap {
+  margin-top: 0;
+}
+
+.flourish-pullquote {
+  height: auto;
+  
+  &__text {
+    display: inline-block;
+    font-weight: 400;
+    font-style: italic;
+    font-size: 20px;
+  }
```

Comment:
I feel like we should stick with the gravity `blockquote` component or if it needs to be adjusted then that can be overridden here. Maybe one for @james-nash and @ThePeach to discuss.

Reply:
Used `blockquote` element but that's all as flourish components are only on the website, not within Gravity.